addon/components/phone-input.js

```diff
@@ -136,24 +147,24 @@ export default class PhoneInput extends Component {
     }
     this._iti = _iti
 
+    if (this.initialCountry) {
+      this._iti.setCountry(this.initialCountry)
+    }
+
     this.update(number, this._metaData(_iti))
   }
 
   // this is a trick to format the number on user input
   didRender() {
     this._super(...arguments)
 
-    if (!this._iti) {
```

Comment:
Don't we need this guard to make sure that the `intl-tel-input` is setup before trying to update it?

Reply:
I'll revert, you're right